### ISSUE TITLE
Suggestions for incorporating one-time payments and subscriptions?

### ISSUE THREAD

**Issue by sgbaird** [saasforge/open-source-saas-boilerpate#39]
Either incorporating into this repo or a suggestion for an alternative:

Also interested in:
- "purchase credits" models as in [DALLE-2](https://openai.com/dall-e-2/) (purchase credits, use credits over time)

Lmk if I just missed it in the docs

**Comment by saasforge** [comment]
No, we currently don't support one-time payments because this boilerplate can be mostly used for development SaaS applications what usually means regular payments.

**Comment by sgbaird** [comment]
@saasforge thanks for clarifying! Could you point me to where in the docs it covers regular payments?

**Comment by sgbaird** [comment]
Ok, I'm seeing the mention of integration with Stripe for paid versions

![image](https://user-images.githubusercontent.com/45469701/194734214-3371fec6-8923-491e-80c8-5c2d9a16661f.png)

**Comment by saasforge** [comment]
The full documentation can be found here on the website:
https://www.saasforge.dev/docs/services/payment-service
https://www.saasforge.dev/docs/modules/payment

But it's not included in the open source version, it's only available for the paid version, see the pricing https://www.saasforge.dev/pricing
